lib/command/command.js

```diff
@@ -432,6 +441,17 @@ internals.isCommandAllowed = function (command, slackResponse, slackData) {
   return true;
 };
 
+internals.isAllowedChannel = function (command, slackResponse, slackData) {
+  if (command && command.allowedChannels) {
+      var getAllChannelsByName =  _.map(slackData.channels, 'name');
+      var allowedChannelList= _.intersection(getAllChannelsByName,command.allowedChannels);
+      var currentActiveChannel = _.find(slackData.channels, ['id', slackResponse.channel]);
```

Comment:
Hi  usubram 
 yes thanks and I can adapt these changes since I am new to node 
But the logic will be based on the channel name and not on the channel id 
Hope that is good for you